examples/basic/app.css

```diff
@@ -29,3 +29,8 @@
   transform: scale(0.5) rotateX(30deg);
   transition: all 150ms ease-in;
 }
+
+.ReactModal__Body--open,
+.ReactModal__Html--open {
```

Comment:
It looks like the `ReactModal__Html--open` class is never being applied to the HTML element (unless it is set explicitly); in fact, this is tested on line 279 of `specs/Modal.spec.js`.  On the other hand, the `ReactModal__Body--open` class is applied to the body by default, which seems potentially inconsistent.

Reply:
At least one example must use this property, I guess.